### ISSUE TITLE
[Meta] Optimize for Tablet

### ISSUE THREAD

**Issue by lime124** [mozilla-mobile/fenix#4498]
### Why/User Benefit/User Problem
- Fenix was released focused on phone use. We need to go back and optimize the UI for tablets (both portrait and landscape).

### Acceptance Criteria (Added by PM. For EPM to track when a Meta feature is done)
-UX completed
-User stories completed
-Strings written and approved
-QA completed
-Localization done

### What / Requirements (Added by PM and Eng Manager)
-UX Designs (Immediate task: Assign UX issue to this epic)
-User stories (to be created by PM)
-List dependencies on other issues/teams etc.

### Reference Issues:
#2344 
#3999

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-12889)

**Comment by lime124** [comment]
@vesta0

**Comment by vesta0** [comment]
This is not a Q3 priority but it might be fun to check out the ['b:tablet'](https://github.com/mozilla-mobile/fenix/issues?q=is%3Aopen+is%3Aissue+label%3Ab%3Atablet) bugs as a starting point.

**Comment by ayushi1998** [comment]
@vesta0  I am an outreachy applicant, new to open source. Should I take up this issue? Kindly assign it to me.

**Comment by vesta0** [comment]
Hi @ayushi1998 thanks for your interest in contributing to the Fenix project! This issue is not ready to be worked on yet but please check out issues with `help wanted` or `good first issue` labels!

**Comment by karlcow** [comment]
Does this include about changing the ua string to be a desktop ua string like safari on ipad.

**Comment by Poopooracoocoo** [comment]
#4134
#4140
#4203

**Comment by abhijitvalluri** [comment]
How is this not yet implemented for tablets? I find it ridiculous and unacceptable that Firefox version 79 was shipped to users without any tablet optimizations! It is a slap in the face of loyal users and an open invitation to go to better browsers. This is how you lose your user base even more!

**Comment by Jeremias-A-Queiroz** [comment]
Hello

I don't know how the github works. I just wanted to know the status of this "feature request". After all, this worked correctly in the previous version and now they have put the 79 into production without taking into account tablet users.

Right now I am trying to use "GNU Icecat" but it cannot correctly replace the FF.

Thank you very much in advance.

**Comment by anotherus3r** [comment]
Yeah, I too miss the tablet-optimized UI, especially the tab row on the top (could also be on the bottom?) of the screen. The new tab page on tablets could also get some additional content, as there's a lot of empty space currently.

**Comment by fawzib** [comment]
i was gonna use firefox and its password manager to replace chrome but you guys are 10 years behind in features and usability. I encountered over dozen bugs in 3 days and another dozen of obvious quality of life missing improvements.

**Comment by Dunexus** [comment]
@fawzib Could you check if there are open tickets about your bugs an quality of life missing improvements ? Feel free to open some to help improving this product.

**Comment by mattboy9921** [comment]
I've been rooting for Firefox to reimplement the tablet UI ever since I used it on my Nexus tablet way back in 2012. It's now 2021 and my daily driver smartphone is a Z Fold 2. It makes me so sad I can't use my favorite browser on my phone because the desktop-like tab bar in Chrome is just so much better. With the foldable market becoming larger now, it makes optimizing for tablets even more relevant.

**Comment by paroj** [comment]
you guys need to understand that this is a tracker for technical issues and not a feedback form.
I guess nobody opposes the idea of having a dedicated tablet UI - I mean we had it pre fenix.
However, given the android tablet marketshare, I can clearly see why the developers first tackling *all other stuff* first.

**Comment by corbindavenport** [comment]
There's plenty of other issues about feature requests. That being said, I think this might need to be merged with https://github.com/mozilla-mobile/fenix/issues/14921.

**Comment by gbil** [comment]
>  However, given the android tablet marketshare, I can clearly see why the developers first tackling _all other stuff_ first.

Talking for myself here but probably for other people, this is not the way to look at it because it was there before and an ecosystem was built from users based on this. I no longer user firefox because that ecosystem was broken when the tablet features were removed in fenix and to be more clear:

I can no longer use in my tablet the desktop site by default, the desktop UI and the keyboard shortcuts so I have to use a different browser on my tablet with these features -> so I need to use that different browser on my laptop and my desktop because I'm in an ecosystem and I sync my devices and I expect to be able to move from one device to another seamless 

To summarize, if this was a new feature to Firefox Mobile then sure, take into account multiple things and tablet usage numbers etc. But this is not a new feature to Firefox Mobile, it was there before and removed so it is about feature parity, anything else is technicalities by the devs and Mozilla

**Comment by dissociativity** [comment]
This does also ignore than chromebook users may opt to use the Firefox browser even if the chromebook offers a desktop version of chromium, as they're using a Chrome os device to have a cheap laptop with Android app compatibility, they can opt to run the native Linux version and get worse battery life and additional layers of complexity, but chromebook are going to tend to default to Android apps, ideally with tablet/desktop-centric ui options.

Now with Windows 11's integration Android app support, this is also relevant for Windows tablet users who would like more touch friendly applications when not using the tablet docked.

**Comment by ActualDisaster2447** [comment]
It's 2022 and there is still no sign of work on a tablet UI. And then people wonder why Firefox is bleeding users...

**Comment by jr-johnson-support** [comment]
CX team here: 

We have had 49 requests for improvements to Tablet UI. Tracking since Jun 2022 and last updated Jan 9th. 

**Feature Description:**
Users are requesting that Firefox for Android becomes optimized for tablet UI. Users have mentioned the following: 
- displaying bookmarks below the address bar 
- matching the desktop UI 

**How does this improve your mobile experience?**
A tablet has a larger screen so users would like Firefox to match more with the desktop version

**Comment by Atemu** [comment]
Note that we don't need an exact match for the desktop UI. That would also be an option but Fennec had its own UI here and I think it was better that way.

What's important is that we have a scrollable tab bar (perhaps even at the bottom?). A tab menu as the main workflow is a no-go on a tablet.  
Everything else is more-or-less fine actually and might only need some small touchups w.r.t. screen size.

**Comment by Poopooracoocoo** [comment]
Would be interesting if some liberties were taken to experiment with the design, much like earlier versions of Fenix, and as Fenix still does (new tabs don't exist!). Internet Explorer for Windows 8 could be used as inspiration.

But personally I'd much prefer to see an interface similar to Fennec, Chrome, Samsung Internet or Safari, etc

**Comment by kbrosnan** [comment]
Moved to bugzilla: https://bugzilla.mozilla.org/show_bug.cgi?id=1812869

Change performed by the [Move to Bugzilla add-on](https://addons.mozilla.org/en-US/firefox/addon/move-to-bugzilla/).
